src/Cpanel/PluginActions.php

```diff
@@ -21,4 +51,51 @@ public function applyDefaultSettings()
     {
         // Do nothing
     }
+
+    public function getConfig()
+    {
+        $this->getUserConfig();
+        $this->getComposerJson();
+
+        //Clone the config to manipulate
+        $config = array_merge(array(), self::$CONFIG);
+
+        $config['version'] = $this->composer['version'];
+
+        // Merge and intersect arrays and return responses
+        $response = array_intersect_key($this->userConfig + $config, $config);
+        return $response;
+    }
+
+    public function getUserConfig()
+    {
+        if ($this->userConfig === null) {
+            //Need to suppress the File not found error with @
+            $userConfigContent = @file_get_contents('./config.json');
+
+            //Need to set an empty array for merge into config
+            if ($userConfigContent === false) {
+                $this->userConfig = [];
+            } else {
+                $this->userConfig = json_decode($userConfigContent, true);
+            }
```

Comment:
`else` case is almost never needed. This can be written as

```
$this->userConfig = [];
if ($userConfigContent) {
   $this->userConfig = json_decode($userConfigContent, true);
}
```